### DISCUSSION TITLE
Convert K-33 to K-32?

### DISCUSSION THREAD

**Discussion by levens-github** [Chia-Network/chia-blockchain#1404] [Q&A] [answered]
I started plotting several on K-33 and don't want to lose the time spent, is there a way to convert o K-32 without starting over?

**Comment by hoffmang9** [comment, marked as answer]
No.

**Reply by levens-github** [reply]
ok thanks, if I am starting over, any reason not to put several plots in the same directory?

**Reply by FunkyPDB** [reply]
Why do you want to start over on k32? I’ve been making my first plot using k33 and 36 hours in I’m still plotting 🤷‍♂️
